restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/SmsInterpreter.java

```diff
@@ -805,28 +807,6 @@ public void execute(final Object message) throws Exception {
             } else {
                 // Start observing events from the sms session.
                 session.tell(new Observe(source), source);
-                // Store the status callback in the sms session.
```

Comment:
@jaimecasero I agree removing the check for callback attribute at this point and especially to not stop the interpreter in case the callback url is wrong. As far as I can see in the DAO classes if the callback url is wrong it will result to null, which to me is the correct way to handle the url at this point.
But here we miss the relsolve of relative urls which might be used in an RVD application (and for other application servers)

Reply:
ummm,this really need careful thinking. Let me find out more details on this.